backend/src/test/java/com/github/JeTSkY1h/AlexandraIntegrationTest.java

```diff
@@ -1,35 +1,53 @@
 package com.github.JeTSkY1h;
 
+import com.cloudinary.Cloudinary;
+import com.cloudinary.http42.api.Response;
+import com.cloudinary.utils.ObjectUtils;
 import com.github.JeTSkY1h.book.Book;
+import com.github.JeTSkY1h.book.CloudinaryService;
 import com.github.JeTSkY1h.user.*;
 import nl.siegmann.epublib.util.IOUtil;
-import org.assertj.core.api.Assert;
 import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.Test;
+import org.mockito.Answers;
+import org.mockito.Mockito;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
 import org.springframework.boot.test.web.client.TestRestTemplate;
 import org.springframework.http.*;
 
 import java.io.BufferedInputStream;
+import java.io.File;
 import java.io.FileInputStream;
 import java.io.InputStream;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 
 import static org.junit.jupiter.api.Assertions.fail;
 
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class AlexandraIntegrationTest {
+
     @Autowired
     TestRestTemplate restTemplate = new TestRestTemplate();
 
+    @MockBean(answer = Answers.RETURNS_DEEP_STUBS)
+    Cloudinary cloudinary;
+
+    @MockBean
+    CloudinaryService cloudinaryService;
```

Comment:
When you mock the `CloudinaryService` you don't need to mock `Cloudinary` anymore, do you? Isn't `Cloudinary` only uses in the `CloudinaryService`?

Reply:
I use this mockbean at line 100.